src/sentry/integrations/vsts/issues.py

```diff
@@ -31,14 +31,14 @@ def get_create_issue_config(self, group, **kwargs):
         project_choices = []
         initial_project = ('', '')
         for project in projects:
-            project_choices.append((project['id'], project['name']))
-            if project['id'] == self.default_project:
+            project_id_and_name = '%s#%s' % (project['id'], project['name'])
```

Comment:
You append join these values together and then split them later on (line 72). Could you use a tuple/dict instead to avoid joining/splitting?

Reply:
@markstory is it possible to use a tuple for the id of a js choice field? Js is not my primary language by any means; I wasn't aware that was possible.

Reply:
No it wouldn't be. I didn't know this was ending up in a clientside picker. My bad.

Reply:
I'm kind of confused why we need to combine project and name in general? is the ID not unique? I may be forgetting context here

Reply:
@EvanPurkhiser Yes the id is unique, but I wanted the name so I didn't have to look it up again right after the user presses submit on the form. Otherwise it requires another api call. If there's a different way to do that do let me know! I can follow up with another pr and make improvements